src/ViewModel/Converter/ArticleContentHeaderConverter.php

```diff
@@ -30,11 +32,27 @@ public function __construct(ViewModelConverter $viewModelConverter, UrlGenerator
      */
     public function convert($object, string $viewModel = null, array $context = []) : ViewModel
     {
+        $isMagazine = $context['isMagazine'] ?? false;
+
+        $breadcrumb = ($isMagazine || 'feature' === $object->getType()) ? [
+            new ViewModel\Link(
+                'Magazine',
+                $this->urlGenerator->generate('magazine')
+            ),
+        ] : [];
+
+        $breadcrumb[] = new ViewModel\Link(
+            ModelName::singular($object->getType()),
+            $this->urlGenerator->generate('article-type', ['type' => $object->getType()])
+        );
+
         $subjects = $object->getSubjects()->map(function (Subject $subject) {
             return new ViewModel\Link($subject->getName(), $this->urlGenerator->generate('subject', [$subject]));
         })->toArray();
 
-        $authors = ($object->getAuthors()->notEmpty()) ? $this->convertTo($object, ViewModel\Authors::class) : null;
+        $authors = (!$isMagazine && $object->getAuthors()->notEmpty()) ? $this->convertTo($object, ViewModel\Authors::class) : null;
+
+        $impactStatement = ($isMagazine && $object instanceof HasImpactStatement) ? $object->getImpactStatement() : null;
```

Comment:
Not sure I fully understand the logic here - are we withholding impact statements from non-magazine types, even if they have one?

Reply:
nvm, it's me.